resources/application.ts

```diff
@@ -25,6 +32,10 @@ export class Applications extends BaseResource {
         return this.httpPost<UnitResponse<Application>>("", { data: request })
     }
 
+    public async createForm(request: CreateApplicationFormRequest): Promise<UnitResponse<CreateApplicationFormResponse> | UnitError> {
+        return this.httpPost<UnitResponse<CreateApplicationFormResponse>>("-forms", { data: request })
+    }
+
```

Comment:
This probably won't work since the resourcePath on this file is "application**s**" and the application forms resource is "application-forms", without the "s".

We consider the application form to be a separate resource, it should therefore be contained in its own file.
If you could, move the application form related code to a new resource file named `applicationForm.ts`

Reply:
understood, i will fix everything according.